CharactersPage.js

```diff
@@ -559,32 +559,26 @@ function inject_join_exit_abovevtt_button() {
   if ($(".ddbc-campaign-summary").length === 0) return;     // we don't have any campaign data
   if ($("#avtt-character-join-button").length > 0) return;  // we already injected a button
 
+  $(".ct-character-sheet-desktop > .ct-character-header-desktop").css({display: "inline-flex"})
```

Comment:
This unfortunately pushes the edit button off the other side of the screen for me when not full screen with beyond20 enabled. 

I think the button overlapping the player header tidbits (xp race etc.) is also only an issue with beyond20 enabled and not full screen?

It's still a good issue to fix since so many of our users use beyond20.

![image](https://user-images.githubusercontent.com/65363489/236179861-25b661d8-71c4-4fba-9de3-1c6531106415.png)